Tests/C++WorkshopTests/CardsTests.mm

```diff
@@ -25,8 +25,8 @@ @interface Cards: XCTestCase
     using Super::begin;
     using Super::end;
     
-    auto draw() -> Card<Deck>& {
-        auto card = this->_cards.front();
+    auto draw() -> Cards::value_type {
```

Comment:
This returns `Cards::value_type`, but elsewhere, we return `std::unique_ptr<Card>;`
Is there a difference?

Reply:
They're the same type; I just wasn't consistent in my usage. I think `Cards::value_type` is a little nicer because it's explicit about the relationship between this return type and the stored type, but maybe that's leaking too much implementation information? 🤷